timemachine/fe/atom_mapping.py

```diff
@@ -202,6 +202,24 @@ def induce_subgraphs_given_marcs(mol_a, mol_b, core, marcs):
     return sg_a, sg_b
 
 
+def induce_subgraphs_given_bond_core(mol_a, mol_b, core, bond_core):
+    sg_a = nx.Graph()
+    sg_b = nx.Graph()
+    for a in core[:, 0]:
+        sg_a.add_node(a)
+    for b in core[:, 1]:
+        sg_b.add_node(b)
+
+    # edges_a = get_edges(mol_a)
+    # edges_b = get_edges(mol_b)
+    for e1, e2 in bond_core.items():
+        src_a, dst_a = e1
+        src_b, dst_b = e2
+        sg_a.add_edge(src_a, dst_a)
+        sg_b.add_edge(src_b, dst_b)
+    return sg_a, sg_b
```

Comment:
Style nit: Could be slightly cleaner(?) to write a function `induce_edge_subgraph(mol, edges)` and call it twice, rather than write 2x duplicated instructions for inducing 2 edge subgraphs in the same function